resources/scripts/artifacts-api-latest-release-versions.sh

```diff
@@ -28,7 +28,8 @@
 set -eo pipefail
 
 URL="https://artifacts-api.elastic.co/v1"
+NO_KPI_URL_PARAMS="x-elastic-no-kpi=true"
```

Comment:
Nit: this is a single parameter, so a better name might be `NO_KPI_URL_PARAM` (without the `S`). That will make it easier to read if we end up needing to add additional parameters.

Reply:
Yep, totally agree. Will change it ASAP